### PR TITLE
Adds the ability to create forks

### PR DESCRIPTION
**Pull request by danielrs**
Adds the ability to create forks and fixes the "is_private" parameter (it's supposed to be boolean and not a string).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/ktrysmt/go-bitbucket
 go 1.12
 
 require (
-	github.com/golang/protobuf v1.0.0
+	github.com/golang/protobuf v1.0.0 // indirect
```
Comment (ktrysmt) [inline go.mod:6]: Would you remove the line if it is not a problem?

File: repository.go [hunk 2]
```diff
@@ -331,7 +343,7 @@ func (r *Repository) buildRepositoryBody(ro *RepositoryOptions) string {
 	//		body["name"] = ro.Name
 	//}
 	if ro.IsPrivate != "" {
-		body["is_private"] = ro.IsPrivate
+		body["is_private"] = strings.ToLower(strings.TrimSpace(ro.IsPrivate)) != "false"
```
Comment (ktrysmt) [inline repository.go:346]: I thought the below was the right one, and do you think?
```suggestion
		body["is_private"] = strings.ToLower(strings.TrimSpace(ro.IsPrivate)) 
```

Comment (danielrs) [reply repository.go:346]: @ktrysmt Bitbucket expects a boolean value in the payload for the `is_private` field. If you try and submit a request with the previous code you get the error:

```
{"type": "error", "error": {"fields": {"is_private": "expected bool"}, "message": "Bad request"}}
```

The new condition of `!=` makes the value a real boolean, which is what Bitbucket expects.

File: go.mod (hunk 1 above)
Comment (danielrs) [reply go.mod:6]: Got it

File: repository.go (hunk 2 above)
Comment (ktrysmt) [reply repository.go:346]: I see.
I understood that `is_private: boolean` is detected at the site.

**Comment by ktrysmt** [comment]
Please see the new release at https://github.com/ktrysmt/go-bitbucket/releases/tag/v0.6.2
Thank you @danielrs .

**Pull request merged**